api/voice-react-native-sdk.api.md

```diff
@@ -347,13 +347,22 @@ export class CallMessage extends EventEmitter {
     // Warning: (ae-forgotten-export) The symbol "NativeCallMessageInfo" needs to be exported by the entry point index.d.ts
     //
     // @internal
-    constructor({ content, contentType, messageType, voiceEventSid, }: NativeCallMessageInfo);
+    constructor(callMessageInfo: NativeCallMessageInfo);
     getContent(): any;
     getContentType(): string;
     getMessageType(): string;
     getSid(): string | undefined;
 }
 
+// @public
+export namespace CallMessage {
+    export interface Options {
+        content: any;
+        contentType?: string;
+        messageType: string;

```

Comment:
Why are we wrapping this under CallMessage namespace? We use CallMessage interface on all platforms, which is different than what we're doing here.